workflow/main.py

```diff
@@ -0,0 +1,213 @@
+'''
+Main driver code for end to end TPS workflow
+
+@Author Adam Shedivy 
+
+'''
+
+from visualization_utilities import vis, find_path, process_exists
+from table_generation import PrepTemporalCytoscapeTPS
+from table_refactor import refactor_col_delim_heat, refactor_col_delim
+from requests.exceptions import ConnectionError as CE
+from json.decoder import JSONDecodeError
+import os
+import pandas as pd
+import numpy as np
+from datetime import date
+import subprocess
+import sys
+import time
+import csv
+
+
+def main():
```

Comment:
This is a fairly long function.  Are there places where it can be split into smaller components?

Reply:
There are 3 main sections to the main function: 

- grab all of the input files, run annotations generation
- run Cytoscape
- Run visualization scripts

These can all be modularized further for better organization. Once we start using a config file, the code will be cleaned up tremendously.